frontend/src/bundles/common/components/bar-chart/config-bar.ts

```diff
@@ -0,0 +1,33 @@
+const options = {
+    scales: {
+        y: {
+            ticks: {
+                callback: (value: {
+                    toLocaleString: (
+                        argument0: string,
+                        argument1: { style: string; currency: string },
+                    ) => string;
+                }): string =>
+                    `+${value
+                        .toLocaleString('en-US', {
+                            style: 'currency',
+                            currency: 'USD',
+                        })
+                        .slice(1)} $`,
```

Comment:
It's hard to understand what's going on here.
Create a separate function for the callback and interfaces to it (maybe interfaces can be imported from the package).